### PR TITLE
Refactored code issues

### PR DESCRIPTION
**Pull request by hasuniea**

### CODE REVIEW

File: components/device-mgt/org.wso2.carbon.device.mgt.mobile.impl/src/main/java/org/wso2/carbon/device/mgt/mobile/impl/windows/util/WindowsUtils.java [hunk 1]
```diff
@@ -18,18 +18,32 @@
...
+        mobileDevice.setImsi(rs.getString(WindowsPluginConstants.IMSI));
+        mobileDevice.setModel(rs.getString(WindowsPluginConstants.DEVICE_MODEL));
+        mobileDevice.setVendor(rs.getString(WindowsPluginConstants.VENDOR));
+        mobileDevice.setLatitude(rs.getString(WindowsPluginConstants.LATITUDE));
+        mobileDevice.setLongitude(rs.getString(WindowsPluginConstants.LONGITUDE));
+        mobileDevice.setSerial(rs.getString(WindowsPluginConstants.SERIAL));
+        mobileDevice.setOsVersion(rs.getString(WindowsPluginConstants.LATITUDE));
+        return mobileDevice;
+    }
+    public static Feature getMobileFeature() {
```
Comment (prabathabey) [inline components/device-mgt/org.wso2.carbon.device.mgt.mobile.impl/src/main/java/org/wso2/carbon/device/mgt/mobile/impl/windows/util/WindowsUtils.java]: Need a new line between the boundaries of these two methods

Comment (hasuniea) [inline components/device-mgt/org.wso2.carbon.device.mgt.mobile.impl/src/main/java/org/wso2/carbon/device/mgt/mobile/impl/windows/util/WindowsUtils.java]: fixed the issue from https://github.com/wso2/carbon-device-mgt-plugins/pull/181

File: components/device-mgt/org.wso2.carbon.device.mgt.mobile.impl/src/main/java/org/wso2/carbon/device/mgt/mobile/impl/windows/util/WindowsUtils.java [hunk 2]
```diff
@@ -18,18 +18,33 @@
...
+        mobileDevice.setModel(rs.getString(WindowsPluginConstants.DEVICE_MODEL));
+        mobileDevice.setVendor(rs.getString(WindowsPluginConstants.VENDOR));
+        mobileDevice.setLatitude(rs.getString(WindowsPluginConstants.LATITUDE));
+        mobileDevice.setLongitude(rs.getString(WindowsPluginConstants.LONGITUDE));
+        mobileDevice.setSerial(rs.getString(WindowsPluginConstants.SERIAL));
+        mobileDevice.setOsVersion(rs.getString(WindowsPluginConstants.LATITUDE));
+        return mobileDevice;
+    }
+    
+    public static Feature getMobileFeature() {
```
Comment (prabathabey) [inline components/device-mgt/org.wso2.carbon.device.mgt.mobile.impl/src/main/java/org/wso2/carbon/device/mgt/mobile/impl/windows/util/WindowsUtils.java]: This method is redundant, I believe.

Comment (hasuniea) [inline components/device-mgt/org.wso2.carbon.device.mgt.mobile.impl/src/main/java/org/wso2/carbon/device/mgt/mobile/impl/windows/util/WindowsUtils.java]: thats why I just created separate util method to get MobileDevice.

Comment (prabathabey) [inline components/device-mgt/org.wso2.carbon.device.mgt.mobile.impl/src/main/java/org/wso2/carbon/device/mgt/mobile/impl/windows/util/WindowsUtils.java]: Umm. I don't think I understood it. What I wanted to highlight is that, it's redundant having a separate method just to do some regular instantiation of an object. Instead, we're better of merging the corresponding logic inline with wherever this method is currently being used.

File: components/device-mgt/org.wso2.carbon.device.mgt.mobile.impl/src/main/java/org/wso2/carbon/device/mgt/mobile/impl/android/AndroidDeviceManager.java [hunk 3]
```diff
@@ -101,13 +101,13 @@ public boolean saveConfiguration(TenantConfiguration tenantConfiguration)
             status = true;
         } catch (MobileDeviceMgtPluginException e) {
             throw new DeviceManagementException(
-                    "Error occurred while retrieving the Registry instance : " + e.getMessage(), e);
+                    "Error occurred while retrieving the Registry instance : ", e);
```
Comment (prabathabey) [inline components/device-mgt/org.wso2.carbon.device.mgt.mobile.impl/src/main/java/org/wso2/carbon/device/mgt/mobile/impl/android/AndroidDeviceManager.java]: Why colons at the end of the comments? That's not needed, I believe. Fix the same in other places as well.

Comment (hasuniea) [inline components/device-mgt/org.wso2.carbon.device.mgt.mobile.impl/src/main/java/org/wso2/carbon/device/mgt/mobile/impl/android/AndroidDeviceManager.java]: fixed the issue from https://github.com/wso2/carbon-device-mgt-plugins/pull/181

**Pull request merged**